pcmanfm/translations/pcmanfm-qt_es.desktop

```diff
@@ -0,0 +1,4 @@
+#Translations
+Name[es]=Gestor de archivos PCManFM
```

Comment:
PCManFM-qt ?

Reply:
I thought the same, but I didn’t know whether it was appropriate because this same issue was present in [`pcmanfm/pcmanfm-qt.desktop.in`](https://github.com/lxde/pcmanfm-qt/blob/8ba67fdfc8d848e1882f992ee37eb8d795e857dd/pcmanfm/pcmanfm-qt.desktop.in#L3) since 2014. Well, here you go then ...